src/components/Borrow/BorrowTable/BorrowTable.tsx

```diff
@@ -125,13 +125,14 @@ export const BorrowTable: React.FC<IProp> = ({
                 <Grid classes={{ root: classNames(classes.rootCell, classes.amountColumn) }}>
                   <Tooltip
                     classes={{ tooltip: classes.tooltipNumber }}
-                    title={`${element.deposited} ${element.collateral}`}
+                    title={`${element.deposited.toFixed(7)} ${element.collateral}`}
                     placement='bottom-start'>
                     <Grid>
                       {'~ '}
                       <AnimatedNumber
-                        value={element.deposited}
+                        value={element.deposited.toFixed(6)}
                         formatValue={formatNumbersBorrowTable}
```

Comment:
Imo this should be to fixed decimals of assets

Reply:
ok, changed